staking/cli/src/instructions.rs

```diff
@@ -0,0 +1,472 @@
+
+    //decode tmp from base64
+    let message = base64::prelude::BASE64_STANDARD
+        .decode(encoded_message)
+        .unwrap();
+
+    let (vaa, merkle_proofs) = deserialize_accumulator_update_data(message);
+
+
+    let encoded_vaa = process_write_encoded_vaa(rpc_client, vaa.as_slice(), wormhole, payer);
+
+
+    let publisher_caps = Keypair::new();
```

Comment:
`init_publisher_caps` could return the pubkey for `publisher_caps`

Reply:
done.